packages/api/src/schema/types/shop.js

```diff
@@ -4,13 +4,24 @@ export default [
       href: String
       title: String
     }
+
+    type CustomEntityInterface {
```

Comment:
the type name should be something like AdminUiConfigCustomEntityInterface (prefixed with admin ui), else "CustomEntityInterface" is too general